librz/asm/asm.c

```diff
@@ -1323,7 +1323,10 @@ RZ_API RZ_OWN RzAsmTokenString *rz_asm_token_string_clone(RZ_OWN RZ_NONNULL RzAs
 	new->str = rz_strbuf_new(rz_strbuf_get(toks->str));
 	new->op_type = toks->op_type;
 
-	rz_return_val_if_fail(new->tokens &&new->str, NULL);
+	if (!(new->tokens &&new->str)) {
```

Comment:
not formatted.

Reply:
The CI rejected the space in `if (!(new->tokens && new->str)) {`.

Reply:
what, that is weird.

Reply:
i know why, rename that `new` pointer to something else. `new` is a keyword and should not be used.

Reply:
Ah, right. Thanks